src/doc/en/developer/coding_basics.rst

```diff
@@ -165,6 +165,16 @@ of the following places:
 
     os.path.join(os.path.dirname(__file__), 'sage-maxima.lisp')
 
+- Large data files should not be added to the Sage source tree.
+  Instead:
+
+  - Create a separate git repository for them
+  - Add metadata in your repository that make it a pip-installable
+    package (distribution package)
+  - Upload it to PyPI
+  - Create metadata in ``SAGE_ROOT/build/pkgs`` that make your new
+    pip-installable package known to Sage
+
```

Comment:
Nice. Some ideas:
 - give https://github.com/sagemath/conway-polynomials and https://github.com/gmou3/matroid-database as examples
 - suggest compressing large data files with xz, read transparently as in https://github.com/sagemath/conway-polynomials/blob/master/src/conway_polynomials/__init__.py#L116
 - suggest trying `xz -T 0 -e` to compress, sometimes gives big gains for highly repetitive text files (without affecting the cost for decompression).

Reply:
Thanks! Yes, linking to these projects as examples of best practices is a good idea. Will do (unless someone sends me a PR first... hint, hint)

Reply:
@tornaria are these points taking care of by the current version of the text already?